### ISSUE TITLE
Capacitive KeyPad without mouse pointer?

### ISSUE THREAD

**Issue by TommelTwo** [phhusson/unihertz_titan#3]
In lineage-17.1-20200808-UNOFFICIAL-treble_arm64_bvN.img I would like to activate the capacitive KeyPad for scrolling. I could successfully deactivate the mouse pointer but I would like to have the scroll function again. Is there already a solution for this?

**Comment by phhusson** [comment]
Yeah you can check in git history. before disabling it, it did scrolling
(though not really great)

**Comment by mathewjpotter** [comment]
hey how did you remove the cursor, also any way to make the top drop down menu full width

**Comment by TommelTwo** [comment]
Here is the solution: https://docs.google.com/document/d/1Kj7poftuXIb9ZWAHxNzDCjwbCYwGQnohe7dV4EYnkQk/edit?usp=sharing

**Comment by TommelTwo** [comment]
the mouse pointer is gone and unfortunately the scroll function too ...
